pgconn.go

```diff
@@ -62,6 +76,40 @@ func (pgdb *PostgresDB) Reconnect(retryCount int) error {
 	return nil
 }
 
+func (pgdb *PostgresDB) SetMaxOpenConns() {
```

Comment:
make both method package access level,
developers can call SetMaxOpenConns(int) if they want to explicit configure